### DISCUSSION TITLE
XRechnung und Umsatzsteuerfreier Artikel

### DISCUSSION THREAD

**Discussion by TietjenMichael** [stephanstapel/ZUGFeRD-csharp#325] [General]
Hallo Community,
ich bekomme folgende Konstellation in einer XRechnung nicht abgebildet

Artikel 1 - Ust 19% - Netto 10,00
Artikel 2 - Ust 19% - Netto 20,00
Artikel 3 - **Ust 0%** - Netto=Brutto 50.00 (Artikel ist Ust befreit )

---- TOTAL  ----

Rechnungssumme 
Netto 30,00
plus Ust 19% = 5,70
= Brutto 35,70
**plus Ust Frei 50,00 (Diese Position bekomme ich nicht abgebildet)**

Gesamt Rechnungsbetrag 85,70

Bekomme leider die Fehler das Netto/ Brutto Summen nicht mit Summen der Einzelpositionen zusammenpassen.

Kann jemand helfen ?

Vielen Dank 
Michael

**Comment by AndreasWaning** [comment]
Could it be that there is a rounding error? I had that at the beginning

**Reply by TietjenMichael** [reply]
Müsste ich nach meinem Urlaub mal testen, geht aber hauptsächlich darum dass ich den Ust freien Betrag nicht zuordnen kann.
Aber trotzdem vielen Dank

**Comment by AndreasWaning** [comment]
Bei mir geht das mit Umsatz Steuer 0.
Hast du
TaxCategoryCodes.E
gesetzt ?

               if (lineTotalAmountNullMwst > 0)
               {

                   desc.AddApplicableTradeTax(basisAmount: (decimal)lineTotalAmountNullMwst,
                                            percent: 0m,
                                            typeCode: TaxTypes.VAT,
                                            categoryCode: TaxCategoryCodes.E,
                                            null,
                                            null,
                                            "steuerfrei"
                                            );
                   
               }

**Reply by TietjenMichael** [reply]
Hallo, den einzelnen Ust freien Artikel habe ich mit Exempt from Tax angegeben. Das funktioniert auch alles.
Was nicht funktioniert ist die Angabe der Summe der Ust freien Artikel bei den Total Angaben:
Netto - ok
Mwst 19% - ok
Brutto - ok
Ust freie Summe. - nicht ok

**Reply by IsThatUniqueEnough** [reply]
IMHO gehören die 50€ mit in den Brutto- und Nettobetrag.

**Reply by TietjenMichael** [reply]
Laut Finanzamt leider nicht.
Ich habe zb
Netto 100.00
Mwst 19.00
Brutto 119.00
Ust frei 50.00 *
————————-
Re Total 169.00

* bekomme ich nicht in die Erechnung

**Reply by IsThatUniqueEnough** [reply]
M.E. gibt es in der XRechnung in SpecifiedTradeSettlementHeaderMonetarySummation keine weitere Summe für steuerfreie Beträge. Welche Beträge mit welchen Steuersätzen welchen Steuerbetrag ergeben, hat man ja auch schon in ApplicableHeaderTradeSettlement/ApplicableTradeTax aufgedröselt. In SpecifiedTradeSettlementHeaderMonetarySummation werden dann nur noch die Netto-, Brutto- und Steuerbeträge aufsummiert, unabhängig vom Steuersatz.

**Reply by TietjenMichael** [reply]
Danke, werde ich mal so meinem Auftraggeber weitergeben.
